README.md

```diff
@@ -1,31 +1,61 @@
-### ReScience C published articles
+## Publishing a ReScience C article
 
-In order to publish a new article (after acceptance), you'll need to have the
-article metadata file (YAML format) and the corresponding article (PDF
-format). The metadata file should be missing the article DOI, number and
-URL. The first step is thus to request this information from Zenodo. Before
+This document is intended for ReScience editors. Congrats on accepting a ReScience article for publication! If you're new at editing, this document will guide you through all the necessary steps. If you are a seasoned editor, this will be a handy reference to any steps you may have forgotten. Follow the steps below to complete the metadata, submit to Zenodo, and get the final article published.
+
+### What you will need to begin
+- The article metadata file (`metadata.yaml`) from the author's repository.   At this time the metadata file should **will be missing** the article DOI, number and URL.  This is expected and something you will fix over the coming steps.
+- The final article itself (`article.pdf`)
```

Comment:
This isn't really the final article, because the DOI is still missing. Pre-final? Sounds weird. Maybe "article as accepted by the reviewers"?